### PR TITLE
Team update

### PR DESCRIPTION
**Pull request by ebressert**
Added some new team members to the AA team.

### CODE REVIEW

File: _data/teams.yml [hunk 1]
```diff
@@ -303,285 +303,274 @@
...
+      twitter:     johnvmcdonnell
+      instagram:
+      flickr:
+      links:
+
+    - name:        Richie 
+      lastname:    Li 
+      location:    San Francisco
+      division:
+      description: Richie likes machine learning, crunching data and writing Java code. He enjoys learning new technologies and tools. During his free time he loves cooking, video gaming and hiking. Hails from the south west of China.
```
Comment (simeonwillbanks) [inline _data/teams.yml]: `south west` is spelled `southwest` at [line 542](https://github.com/stitchfix/stitchfix.github.io/pull/53/files#diff-4dfacf056472ef2c8c2fd870b10c632bR542).  Why don't we pick one spelling for both cases? Thanks!

Comment (davidamcclain) [inline _data/teams.yml]: `southwest` is correct, I believe. But I'm not from the `southwest` so how would I know?

Comment (ebressert) [inline _data/teams.yml]: I've changed the variants to southwest, which seems to be the most common way to write it.

**Comment by ebressert** [comment]
If the update looks good, could someone merge the PR?

**Pull request merged**